packages/e2e-tests/jest.config.js

```diff
@@ -1,7 +1,13 @@
 module.exports = {
 	...require( '@wordpress/scripts/config/jest-e2e.config' ),
-	setupTestFrameworkScriptFile: '<rootDir>/config/setup-test-framework.js',
 	setupFiles: [
 		'<rootDir>/config/gutenberg-phase.js',
 	],
+	setupFilesAfterEnv: [
+		'<rootDir>/config/setup-test-framework.js',
+	],
+	transformIgnorePatterns: [
+		'node_modules',
+		'scripts/config/puppeteer.config.js',
```

Comment:
This is pretty strange but without this being explicitly set everything explodes in `@babel/core` at this line:

https://github.com/babel/babel/blob/master/packages/babel-core/src/config/files/plugins.js#L145-L149